cognitiveservices/qnamaker_create_kb.go

```diff
@@ -0,0 +1,198 @@
+package main
+
+import (
+	"context"
+	"fmt"
+	"github.com/Azure/azure-sdk-for-go/services/cognitiveservices/v4.0/qnamaker"
+	"github.com/Azure/go-autorest/autorest"
+	"log"
+	"time"
+)
+
+// Replace this with a valid subscription key.
+var subscription_key string = "INSERT KEY HERE"
+
+// Replace this with the endpoint for your subscription key.
+var endpoint string = "https://westus.api.cognitive.microsoft.com"
```

Comment:
Let's use westcentral because that is the region for the free tier

```suggestion
var endpoint string = "https://westcentralus.api.cognitive.microsoft.com"
```